src/components/MetadataCollections/MetadataCollections.test.js

```diff
@@ -1,89 +1,125 @@
-import { BrowserRouter as Router } from 'react-router-dom';
-import { noop } from 'lodash';
+import { MemoryRouter } from 'react-router-dom';
 
 import { StripesContext, useStripes } from '@folio/stripes/core';
-import { StripesConnectedSource } from '@folio/stripes/smart-components';
-import { render } from '@folio/jest-config-stripes/testing-library/react';
+import { screen, within } from '@folio/jest-config-stripes/testing-library/react';
+import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
 
 import withIntlConfiguration from '../../../test/jest/helpers/withIntlConfiguration';
 import metadatacollections from '../../../test/fixtures/metadatacollections';
 import mdSources from '../../../test/fixtures/tinyMetadataSources';
 import MetadataCollections from './MetadataCollections';
 
+jest.mock('react-virtualized-auto-sizer', () => ({ children }) => children({ width: 1920, height: 1080 }));
+
 const tinySources = { mdSources };
 
-const testCollection = {
-  logger: { log: noop },
-  mutator: { collections: {}, mdSources: {}, query: {}, resultCount: {} },
-  props: { history: {}, location: {}, match: {}, staticContext: undefined, children: {} },
-  recordsObj: {},
-  resources: {
-    collections: {},
-    mdSources: { tinySources },
-    query: { query: '', filters: 'permitted.yes,selected.yes', sort: 'label' },
-    resultCount: 30
-  }
-};
-
-const connectedTestCollection = new StripesConnectedSource(testCollection.props, testCollection.logger, 'collections');
-
-const renderMetadataCollections = (stripes) => (
-  render(withIntlConfiguration(
-    <Router>
-      <StripesContext.Provider value={stripes}>
-        <MetadataCollections
-          contentData={metadatacollections}
-          collection={connectedTestCollection}
-          filterData={tinySources}
-          onNeedMoreData={jest.fn()}
-          queryGetter={jest.fn()}
-          querySetter={jest.fn()}
-          searchString="permitted.yes,selected.yes"
-          selectedRecordId=""
-          onChangeIndex={jest.fn()}
-        />
-      </StripesContext.Provider>
-    </Router>
-  ))
+const renderMetadataCollections = (stripes, data) => withIntlConfiguration(
+  <MemoryRouter>
+    <StripesContext.Provider value={stripes}>
+      <MetadataCollections
+        contentData={data}
+        filterData={tinySources}
+        onNeedMoreData={jest.fn()}
+        queryGetter={jest.fn()}
+        querySetter={jest.fn()}
+        searchString="permitted.yes,selected.yes"
+        selectedRecordId=""
+        visibleColumns={['label', 'mdSource', 'permitted', 'freeContent']}
```

Comment:
`<MetadataCollections>` does not define a `visibleColumns` prop

Reply:
Removed `visibleColumns`